UmrechnenMultiGrenzen/form.json

```diff
@@ -0,0 +1,50 @@
+{
+	"elements": [{
+			"type": "Label",
+			"caption": "The following borders exist more than once",
+			"visible": false
+		},
+		{
+			"type": "List",
+			"name": "CalculationData",
+			"caption": "Calculations",
+			"add": true,
+			"delete": true,
+			"sort": {
+				"column": "Border",
+				"direction": "ascending"
+			},
+			"columns": [{
+					"caption": "Border",
```

Comment:
Ich finde die Bezeichnung "Ab" wie es im Ticket steht geschickter als das (ursprüngliche) "Grenze". Das macht insbesondere in der Listendarstellung deutlich, dass die Grenze die untere Grenze darstellt.